### PR TITLE
Feature/add organization view

### PR DESCRIPTION
**Pull request by frenchbread**
Closes https://github.com/Digipalvelutehdas/APIKA/issues/37

### CODE REVIEW

File: organizations/client/add/add.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  <div class="container">
+    {{# autoForm collection=OrganizationsCollection id="addOrganizationForm" type="insert" }}
+      <fieldset>
+        <legend>Add an Organization</legend>
+        {{> afQuickField name='name' }}
+        {{> afQuickField name='url' }}
+        {{> afQuickField name='description' }}
+        <div class="panel panel-default">
+          <div class="panel-heading">
+            Contact
```
Comment (brylie) [inline organizations/client/add/add.html]: Add i18n token here.

File: organizations/client/add/add.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+          </div>
+          <div class="panel-body">
+            {{> afQuickField name='contact.person' }}
+            {{> afQuickField name='contact.phone' }}
+            {{> afQuickField name='contact.email' }}
+          </div>
+        </div>
+      </fieldset>
+      <button type="submit" class="btn btn-primary">
+        Create organization
```
Comment (brylie) [inline organizations/client/add/add.html]: Add i18n token here.

File: organizations/client/add/add.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<template name="addOrganization">
+  <div class="container">
+    {{# autoForm collection=OrganizationsCollection id="addOrganizationForm" type="insert" }}
+      <fieldset>
+        <legend>Add an Organization</legend>
```
Comment (brylie) [inline organizations/client/add/add.html]: Add i18n token here.

File: core/lib/i18n/en.i18n.json [hunk 4]
```diff
@@ -552,6 +552,29 @@
                   }
               }
           }
+      },
+      "organizations": {
+          "name": {
+            "label": "Name"
+          },
+          "url": {
+            "label": "URL super"
```
Comment (brylie) [inline core/lib/i18n/en.i18n.json]: What does 'super' mean here?

**Comment by brylie** [comment]
In general, how do users get to the 'add organization' page?

Also, it might be good to make this autoform appear in a modal dialogue, rather than a route. Then, the modal dialogue can be designed as an add/edit form, so we only maintain one template.

**Comment by jykae** [comment]
@frenchbread @brylie How close is this to be merged? 
I would like to include logo template for add organization view #43

**Comment by frenchbread** [comment]
@jykae @brylie I'll finish it today. Had an issue with labels not displaying correctly, now it's fine. 👍

File: core/lib/i18n/en.i18n.json (hunk 4 above)
Comment (frenchbread) [inline core/lib/i18n/en.i18n.json]: For testing labels. Removing..

**Comment by frenchbread** [comment]
Please review 👁

**Pull request merged**